runtime/vm/KeyHashTable.c

```diff
@@ -501,7 +522,6 @@ hashClassTableStartDo(J9ClassLoader *classLoader, J9HashTableState *walkState, U
 	} else {
 		continueToNext = FALSE;
 	}
-
 	
```

Comment:
While you're here, please remove the trailing whitespace.

Reply:
I'll remove the trailing whitespace on line 525.

There is also trailing whitespace after a few of the `if` statements in this file. We can remove those in a separate PR since this revival work doesn't touch those lines/functions otherwise.

Reply:
Had you deleted line 505 instead of line 504, we wouldn't be having this discussion. ;-) I think it entirely reasonable to fix nearby whitespace issues as part of other work.

I'm fine either way: remove the whitespace on lines you didn't change, or defer that to a separate pull request.